packages/main/src/ColorPicker.js

```diff
@@ -316,9 +316,14 @@ class ColorPicker extends UI5Element {
 	}
 
 	_handleHEXChange(event) {
-		const newValue = event.target.value.toLowerCase();
+		let newValue = event.target.value.toLowerCase();
 		const hexRegex = new RegExp("^[<0-9 abcdef]+$");
 
+		// Shorthand Syntax
```

Comment:
Now, If you type in the input, the short syntax is handled properly, but if you for example use the API to change the color, the previous error persists.

```js
$0.color = "#333"
```